.github/ISSUE_TEMPLATE/Issue-report.yml

```diff
@@ -0,0 +1,134 @@
+        * If still experiencing the issue, please provide as many details as possible below about your hardware, computer setup and code.
+  - type: input
+    id: Board
+    attributes:
+      label: Board
+      description: On which Board does this issue occur?
+      placeholder: eg. ESP32 Dev Module, ESP32-S2, LilyGo TTGO LoRa32...
+    validations:
+      required: true
+  - type: textarea
+    id: devboard
+    attributes:
+      label: Device Description
+      description: What development board or other hardware is the chip attached to?
+      placeholder: ex. DevKitC, plain module on breadboard, etc. If your hardware is custom or unusual, please attach a photo.
+    validations:
+       required: true
+  - type: textarea
+    id: other-hw
+    attributes:
+      label: Hardware Configuration
+      description: Is anything else attached to the development board?
+      placeholder: ex. GPIO 18 & 19 are connected to I2C devices.
+    validations:
+      required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: What version of Arduino ESP32 are you running? If possible, consider updating to the latest version.
+      options:
+        - latest master
+        - v2.0.1
+        - v2.0.0
+        - v1.0.6
+        - other
+    validations:
+      required: true
+  - type: input
+    id: IDE
+    attributes:
+     label: IDE Name
+     description: What IDE are you using?
+     placeholder: eg. Arduino IDE, PlatformIO, IDF component...
+    validations:
+     required: true
+  - type: input
+    id: os
+    attributes:
+      label: Operating System
+      description: On which OS does this issue occur?
+      placeholder: ex. macOS 12.1, Windows 10...
+    validations:
+      required: true
+  - type: input
+    id: Flash
+    attributes:
+      label: Flash frequency
+      description: What flash frequency is used?
+      placeholder: eg. 40Mhz
+    validations:
+      required: true
+  - type: dropdown
+    id: PSRAM 
+    attributes:
+      label: PSRAM enabled
+      description: Is PSRAM enabled?
+      options:
+        - 'yes'
+        - 'no' 
+    validations:
+      required: true
+  - type: input
+    id: Upload
+    attributes:
+      label: Upload speed
+      description: What upload speed is used?
+      placeholder: eg. 115200
+    validations:
+      required: true
+  - type: textarea
+    id: Description
+    attributes:
+      label: Description
+      description: Please describe your problem here and expected behaviour
+      placeholder: ex. Can't connect/weird behaviour/wrong function/missing parameter..
+    validations:
+      required: true
+  - type: textarea
+    id: sketch
+    attributes:
+      label: Sketch
+      description: Please provide your sketch/code which was run
+      placeholder: ex. related part of the code
+    validations:
+     required: true
+  - type: textarea
```

Comment:
adding `render: shell` would format this nicely into a codeblock. Long output logs are more manageable this way

Reply:
Added to both, sketch and Debug message.